tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -270,8 +275,8 @@ func parseManifestListForSnapshotIDs(output string) []string {
 }
 
 // waitUntilServerStarted returns error if the Kopia API server fails to start before timeout
-func (ks *KopiaSnapshotter) waitUntilServerStarted(ctx context.Context, addr string) error {
-	statusArgs := append([]string{"server", "status", "--address", fmt.Sprintf("http://%s", addr)})
+func (ks *KopiaSnapshotter) waitUntilServerStarted(ctx context.Context, addr string, serverStatusArgs ...string) error {
+	statusArgs := append([]string{"server", "status", "--address", addr}, serverStatusArgs...)
 	if err := retry.PeriodicallyNoValue(ctx, 1*time.Second, 180, "wait for server start", func() error {
```

Comment:
```suggestion
	if err := retry.PeriodicallyNoValue(ctx, retryInterval, retryCount, "wait for server start", func() error {
```
And like Nick suggested, we can make `wait for server start` a const as well.